src/public/sass/pages/_confirmation-page.scss

```diff
@@ -0,0 +1,18 @@
+.print-summary-body {
+    display: none;
+}
+
+@media print {
```

Comment:
Can you do something similar to:

```
@include govuk-media-query($media-type: print) {
     .govuk-\!-display-none-print {
       display: none !important;
     }
   }
```

but for screens. Then you just tags elements that should be hidden with one simple class.